### PR TITLE
Clean

### PR DESCRIPTION
**Pull request by wuminzhe**

### CODE REVIEW

**Review by fewensa — APPROVED** [review]

File: docs/high-level-overview.md [hunk 1]
```diff
@@ -1,165 +0,0 @@
-# High-Level Bridge Documentation
```
Comment (AsceticBear) [inline docs/high-level-overview.md:1]: I am not sure if we should keep those docs, some of them are useful IMO.

**Review by HackFisher — APPROVED** [review]
LGTM， but there are conflicts

File: docs/high-level-overview.md (hunk 1 above)
Comment (HackFisher) [reply docs/high-level-overview.md:1]: The deleted docs can still be found from upstream.

OK to delete them before rewriting.

Comment (AsceticBear) [reply docs/high-level-overview.md:1]: Make sense.

**Review by AurevoirXavier — APPROVED** [review]
And I suggest creating a new repo later.

File: docs/high-level-overview.md (hunk 1 above)
Comment (wuminzhe) [reply docs/high-level-overview.md:1]: the docs will be modified and added later.

**Comment by AsceticBear** [comment]
Can we merge this?

**Comment by wuminzhe** [comment]
> Can we merge this?

AurevoirXavier suggest creating a new repo.

**Comment by AsceticBear** [comment]
> > Can we merge this?
> 
> AurevoirXavier suggest creating a new repo.

Even if move action needs, that future work. That's has nothing with this pr, we should merge this first.

**Pull request merged**